list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The current implementation directly uses raw SQL queries without any form of pagination or filtering, which can lead to performance issues with large datasets.

Recommendations:
- **Implement Pagination**: Use SQL `LIMIT` and `OFFSET` clauses to paginate results. This will improve performance and user experience.

Example:
```php
$page = isset($_GET['page']) ? (int)$_GET['page'] : 1;
$limit = 10;
$offset = ($page - 1) * $limit;

$query = "SELECT * FROM books LIMIT ? OFFSET ?";
$stmt = $connection->prepare($query);
$stmt->bind_param("ii", $limit, $offset);
$stmt->execute();
$result = $stmt->get_result();
```